manager/controllers/common_test.go

```diff
@@ -0,0 +1,72 @@
+package controllers
+
+import (
+	"app/base/core"
+	"io"
+	"net/http"
+	"net/http/httptest"
+	"testing"
+
+	"github.com/gin-gonic/gin"
+	"github.com/stretchr/testify/assert"
+)
+
+// Content type variables
+var contentTypeCSV = "text/csv"
+var contentTypeJSON = "application/json"
+
+// Init request
+func PrepareRequest(method string, url string, body io.Reader) (w *httptest.ResponseRecorder, req *http.Request) {
+	req, _ = http.NewRequest(method, url, body)
+	return httptest.NewRecorder(), req
+}
+
+// Check if additional header
+func CheckHeader(req *http.Request, contentType *string) {
+	if contentType != nil {
+		contentTypeVal := *contentType
+		req.Header.Add("Accept", contentTypeVal)
+	}
+}
+
+// Create simple request
+func CreateRequest(method string, url string, body io.Reader, contentType *string, handler gin.HandlerFunc) (
+	w *httptest.ResponseRecorder) {
+	w, req := PrepareRequest(method, url, body)
+	CheckHeader(req, contentType)
```

Comment:
what do you think about using `CheckHeader` inside the `PrepareRequest` function?

Reply:
Good idea, I will add that.